src/Pages/ProjectList.tsx

```diff
@@ -4,13 +4,14 @@ import { useProjectRequests } from "../Functions/ProjectRequests";
 import { useCustomToast } from "../Components/Toast";
 import { useAuth0 } from "@auth0/auth0-react";
 import { useEmployeeRequests } from "../Functions/EmployeeRequests";
+import { PaginatedProjectTable } from "@/Components/paginated-table";
 
 function ProjectList() {
   const { getCompanyProjects, archiveProject, updateProject } =
     useProjectRequests();
   const { getEmployeeByEmail } = useEmployeeRequests();
   const { createToast } = useCustomToast();
-  const [projects, setProjects] = useState<Project[]>();
+  const [projects, setProjects] = useState<Project[]>([]);
```

Comment:
How do we want to show if the database is down? Should we have a spinner with a timeout?